common/lib/capa/capa/customrender.py

```diff
@@ -5,8 +5,7 @@
 These tags do not have state, so they just get passed the system (for access to render_template),
 and the xml element.
 """
-
-from .registry import TagRegistry
+# (for etree module:) pylint: disable=no-member
```

Comment:
is this still required with pylint 1.4.1? this bug may be resolved according to https://bugs.launchpad.net/swift3/+bug/1395768 and we updated to 1.4.1 on 1/20.

If this is still required, I'd prefer you do this at the function - or better, line - level.